File: Software_Technologies/17_Exam_Prep/SoftUni_Babies/C#/SoftUniBabies/Models/Baby.cs
namespace SoftUniBabies.Models
{
    using System;
    using System.ComponentModel.DataAnnotations;

    public class Baby
    {
        [Key]
        public int Id { get; set; }

        public string Name { get; set; }

        public string Gender{ get; set; }

        public DateTime Birthday { get; set; }
    }
}

File: Software_Technologies/17_Exam_Prep/SoftUni_Babies/JavaScript/models/baby.js
const Sequelize = require('sequelize');

module.exports = function (sequelize) {
    let Baby = sequelize.define('Baby', {
        name:{
            type: Sequelize.STRING,
            allowNull: false,
        },
        gender:{
            type: Sequelize.STRING,
            allowNull: false,
        },
        birthday:{
            type: Sequelize.DATEONLY,
            allowNull: false,
        }
    },{
        timestamps: false,
    });

    return Baby;
};